### ISSUE TITLE
.nl generation omits last constant symbol

### ISSUE THREAD

**Issue by freem** [freem/asm6f#11]
When telling the assembler to output an .nl file, the last constant will not appear in the file. Labels seem to work alright, though.

Tested with commit hash 3b68946

Example source (`test.asm`):
```
.enum $0000
	item  .db 0
.ende

.org $8000
	lda item
	sta $00
here:
	jmp here
```

Command line used: `asm6f -n test.asm test.nes`

Expected outcome: `test.nes.ram.nl` will have an entry for the "`item`" variable (specifically: `$0000#item#`)

Actual outcome: `test.nes.ram.nl` is empty.

How to confirm: Add another variable after `item`, such as `item2`. The line for `item` will now appear in `test.nes.ram.nl`

Regression notes: I can't recall if this has always been an issue; I've always had more than one RAM variable defined and never thought to check closely.

**Comment by nicklausw** [comment]
[This line](https://github.com/freem/asm6f/blob/f0bd514010d818992c03f4198f09ed702c38d3c9/asm6f.c#L1156) currently has a < where I believe a <= should go. Sorry, I fixed this for lua exports but totally forgot to for regular ones.

**Comment by freem** [comment]
Fixed in commit af3f54df7338bcb141a3e495133ae37e98bd3634.
